### ISSUE TITLE
Pretrainde model

### ISSUE THREAD

**Issue by Chanfeechen** [LalehSeyyed/CheXclusion#3]
Hi,

This is a very interesting work! May I request for a pretrain for your models? 

Thank you!

**Comment by LalehSeyyed** [comment]
We have provided the Conda environment in the same repository for reproducibility purposes. We are not able to share the trained model and the true label and predicted label CSV files of the test set due to the data-sharing agreement. However, we have provided the patient ID per test splits, random seed, and the code. Then, the true label and predicted label CSV files and trained models can be generated by users who have downloaded the data from the original source following the procedure that is described in the “Data availability” session.
